### DISCUSSION TITLE
留言 | 倾旋的博客

### DISCUSSION THREAD

**Discussion by giscus** [Rvn0xsy/rvn0xsy.github.io#18] [General]
# 留言 | 倾旋的博客

单页不会显示在博客列表，会在导航添加一个菜单。

https://payloads.online/message

**Comment by laowangniubi** [comment]
旋佬，你的免杀文章为何隐藏了，很久之前看过，现在工作了想再看一下，发现无了

**Comment by Rvn0xsy** [comment]
你指的是哪一篇？


------------------原始邮件 ------------------

发件人：叶子yezi ***@***.***>
日期：2023-08-04 上午10:12:00
收件人：Rvn0xsy/rvn0xsy.github.io ***@***.***>
抄送：Subscribed ***@***.***>
主题：Re: [Rvn0xsy/rvn0xsy.github.io] 留言 | 倾旋的博客 (Discussion #18)
旋佬，你的免杀文章为何隐藏了，很久之前看过，现在工作了想再看一下，发现无了

**Reply by laowangniubi** [reply]
那个恶意代码逃逸系列文章

**Reply by laowangniubi** [reply]
您好，我刚刚在您的归档中找到了 ，之前的收藏的不知道为何打不开了

**Comment by Rvn0xsy** [comment]
噢噢，是因为域名解绑了。。文章的路由变化了

 ------------------原始邮件 ------------------
发件人：叶子yezi ***@***.***>
日期：2023-08-04 上午10:16:46
收件人：Rvn0xsy/rvn0xsy.github.io ***@***.***>
抄送：倾旋 ***@***.***>, Comment ***@***.***>
主题：Re: [Rvn0xsy/rvn0xsy.github.io] 留言 | 倾旋的博客 (Discussion #18)
您好，我刚刚在您的归档中找到了 ，之前的收藏的不知道为何打不开了

**Reply by laowangniubi** [reply]
好的谢谢您的回复，也谢谢您的文章，文章内的不少知识和实践让我在这次秋招面试中给面试者带来了不错的印象，感谢

**Comment by Rvn0xsy** [comment]
不客气！



------------------原始邮件 ------------------
发件人：叶子yezi ***@***.***>
日期：2023-08-04 上午10:19:16
收件人：Rvn0xsy/rvn0xsy.github.io ***@***.***>
抄送：倾旋 ***@***.***>, Comment ***@***.***>
主题：Re: [Rvn0xsy/rvn0xsy.github.io] 留言 | 倾旋的博客 (Discussion #18)
好的谢谢您的回复，也谢谢您的文章，文章内的不少知识和实践让我在这次秋招面试中给面试者带来了不错的印象，感谢

**Comment by xXiaoPangg** [comment]
师傅收徒吗，想打红队，可是很迷茫。

**Reply by athenahk** [reply]
伸手

**Comment by athenahk** [comment]
关注你好久了，从最初的一叶知安到现在，你和我的一位学长很像，文笔都这么好，而且也都是一步一个脚印走到现在，文章朴实并带着对生活的思考，希望我也能成为像你一样的人。

**Reply by Rvn0xsy** [reply]
谢谢你

**Comment by jiaandjia** [comment]
<h1>太强了</h1>

**Comment by websecer** [comment]
看了师傅的Havoc-Agent-Handler，但是没搞清楚如何使用，如何在havoc生成的时候出现linux选项，github的图我在软件上没找到

**Comment by fenglilinglegeluan** [comment]
2020看过大佬写的文章，一晃4年了，今天翻笔记才想起来，过来看看！！

**Reply by Rvn0xsy** [reply]
常来～
